src/uct/ib/base/ib_iface.c

```diff
@@ -672,32 +675,80 @@ int uct_ib_iface_is_reachable(const uct_iface_h tl_iface,
                               const uct_device_addr_t *dev_addr,
                               const uct_iface_addr_t *iface_addr)
 {
-    uct_ib_iface_t *iface           = ucs_derived_of(tl_iface, uct_ib_iface_t);
-    int is_local_eth                = uct_ib_iface_is_roce(iface);
-    const uct_ib_address_t *ib_addr = (const void*)dev_addr;
-    uct_ib_address_pack_params_t params;
+    const uct_iface_is_reachable_params_t params = {
+        .field_mask         = UCT_IFACE_IS_REACHABLE_FIELD_DEVICE_ADDR |
+                              UCT_IFACE_IS_REACHABLE_FIELD_IFACE_ADDR |
+                              UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING |
+                              UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING_LENGTH,
```

Comment:
No need `UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING | UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING_LENGTH` in this case, then `info_string` and `info_string_length` can be uninitialized at all.
But on other hand, if you define it, then we expect that parameters must be valid.

Reply:
Yes. I also found. It has been removed. Thanks.